extensions/google-workspace/src/components/FileListItem.tsx

```diff
@@ -75,6 +75,20 @@ export default function FileListItem({ file, email }: FileListItemProps) {
               title="Copy File URL"
               shortcut={{ modifiers: ["cmd", "shift"], key: "," }}
             />
+
+            <Action.CopyToClipboard
+              icon={Icon.Clipboard}
```

Comment:
`Icon.Clipboard` is already added for `Action.CopyToClipboard` so no need for them here:

```suggestion
```